### PR TITLE
Refactor IsInvalid to IsValid

### PR DESCRIPTION
**Pull request by igitur**
Refactor IsInvalid to IsValid
Remove public setter from IXLRangeAddress.IsValid

### CODE REVIEW

File: ClosedXML/Excel/Ranges/IXLRangeAddress.cs [hunk 1]
```diff
@@ -13,12 +13,12 @@ public interface IXLRangeAddress
         IXLAddress FirstAddress { get; set; }
 
         /// <summary>
-        /// Gets or sets a value indicating whether this range is invalid.
+        /// Gets or sets a value indicating whether this range is nvalid.
```
Comment (Pankraty) [inline ClosedXML/Excel/Ranges/IXLRangeAddress.cs]: ```nvalid``` - a typo

File: ClosedXML/Excel/Ranges/XLRangeAddress.cs [hunk 2]
```diff
@@ -123,7 +123,7 @@ IXLAddress IXLRangeAddress.LastAddress
             set { LastAddress = value as XLAddress; }
         }
 
-        public bool IsInvalid { get; set; }
+        public bool IsValid { get; set; } = true;
```
Comment (Pankraty) [inline ClosedXML/Excel/Ranges/XLRangeAddress.cs]: Make setter internal?

Comment (igitur) [reply ClosedXML/Excel/Ranges/XLRangeAddress.cs]: `XLRangeAddress` is already an internal class, so the setter is by default internal.

Comment (Pankraty) [reply ClosedXML/Excel/Ranges/XLRangeAddress.cs]: My bad

**Pull request merged**